### ISSUE TITLE
Decoder factory error

### ISSUE THREAD

**Issue by alperenbabagil** [opentok/accelerator-core-android#83]
## New issue checklist

- [x] I have read all of the [`README`](../README.md) 
- [x] I have searched [existing issues](https://github.com/opentok/accelerator-core-android/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

### General information

- Library version(s): 1.0.20
- iOS/Android/Browser version(s): 7.0
- Devices/Simulators/Machine affected: Device ( Samsung Tab A (2016) )
- Reproducible in the demo project? (Yes/No): Yes (But very rare)
- Related issues: No issues related

## Bug report

After a while from video chat started between 2 devices (One tablet, One real device) application crashes.

#### Expected behavior

> Continious video chat

#### Actual behavior

> App crashes after a while (Happened after ~10 secs, ~1 mins,  ~3 mins)

#### Steps to reproduce

> Starting video chat and waiting

#### Crash log? Screenshots? Videos? Sample project?

Logcat logs:

05-29 12:13:20.172 20068-4235/com.xxxx E/rtc: #
                                                                             # Fatal error in /home/tokbox/jenkins/workspace/task-release-otkit-android/project/android/opentok-android-sdk-webrtc/src/main/jni/wrapper/decoder_factory.cc, line 855
                                                                  # last system error: 2
                                                                  # Check failed: 0 == stride  (0 vs. 1)
                                                                  # 
                                                                  #

05-29 12:13:20.173 20068-4235/com.xxxx A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 4235 (MediaCodecVideo)                                                              
                                                                   [ 05-29 12:13:20.175  2338: 2338 W/         ]
                                                                   debuggerd: handling request: pid=20068 uid=10149 gid=10149 tid=4235

## Question or Feature Request

> ...

**Comment by marinaserranomontes** [comment]
Hi @alperenbabagil ,

We are not able to reproduce it. Please, try again with the latest version. We have updated accelerator-core to use the new OpenTok SDK 2.14 client.

Best,
Marina

**Comment by alperenbabagil** [comment]
An improvement about issue: When I set capture size 1920x1080 at publisher crash happens at subscriber Samsung Tab A (2016). 1280x720 is fine.

**Comment by alperenbabagil** [comment]
A second improvement about issue: When I set capture size 1920x1080 at publisher crash happens at subscriber all Samsung Devices. 1280x720 is fine.

**Comment by marinaserranomontes** [comment]
Thanks @alperenbabagil 

Let us take a deep look to this case. As you know, in the OpenTok Capturer by default, we are not supporting this resolution. The higher supported value is HD(1280X720). So, we need to review this scenario where we are using a Custom Video Capturer with Full HD resolution.

I will send you an answer as soon as possible.

Best,
Marina

**Comment by alperenbabagil** [comment]
Thanks @marinaserranomontes,

We need best quality we can get. So we were using 1080p until a month ago actually. An it happens only Samsung devices. Huawei, LG, ASUS and emulator is fine for now. We'll be waiting your answer.

Best,
Alperen Babagil

**Comment by marinaserranomontes** [comment]
Hi @alperenbabagil ,

As we commented, the 1080p resolution is not supported by default in OpenTok Android client. 

We have detected the problem you commented is related to the hardware decoding. The native decoder we are using can only handle even strides. And it seems the frame received has odd strides.
Regarding the AndroidMediaDecoder used in WebRTC, we observe the same behaviour. https://chromium.googlesource.com/external/webrtc/+/master/sdk/android/src/jni/androidmediadecoder.cc#652

Best,
Marina
